README.md

```diff
@@ -49,10 +49,6 @@ dependencies {
 1. Inside the Android Application class, add the following lines to the onCreate() method:
 
    ```java
-   // Java
-   Forethought.INSTANCE.setup("FORETHOUGHT_API_KEY");
-
-   // Kotlin
    Forethought.setup("FORETHOUGHT_API_KEY")
```

Comment:
I think we want to keep `Forethought.INSTANCE` here, and remove the other one?

Reply:
good catch thanks